### ISSUE TITLE
[BUG] Latest ver. 2.13.2 scripts' update issue

### ISSUE THREAD

**Issue by Cade66** [violentmonkey/violentmonkey#1598]
Hi,

Violentmonkey 2.13.2 / Firefox 105.0.2
No issue with previous Violentmonkey 2.13.1

Scripts are updated on Firefox start even though Violentmonkey settings include `Check for script updates every [0] day(s), use 0 to disable `
All scripts are set to` [x] Allow update, then notify: [ ] on [ ] off -- use global setting [x}`

This means scripts are not auto-updated but may be updated manually. This is how it always worked.
I've tried setting scripts' settings to ` [ ] Allow update, then notify: [ ] on [ ] off -- use global setting [x}` : no auto-update on Firefox start BUT manual update is impossible (server not connected, no per-script message.

I've reverted to VM 2.13.1 until this is fixed.

Thanks.

**Comment by tophf** [comment]
I can't reproduce the problem. Maybe some of the scripts are missing a required script or resource and Violentmonkey tries to download it. You can see if it's the case by going to `about:debugging`, then `This Firefox`, then click `Inspect` in the entry for Violentmonkey, and open the console inside - there will be a warning like `The "require" storage is missing "http://foo/bar.js"! Vacuuming...`

**Comment by Cade66** [comment]
You can't reproduce the problem, and I've never encountered it before this ver. 2.13.2 ...

I've checked again with same above mentioned conditions. All caches are cleaned on FF exit.

I start Firefox and connections are established for all 7 scripts to Greasemonkey (all downloaded scripts were installed from there);

**In about:networking:**

![1](https://user-images.githubusercontent.com/65161029/194726049-bca47d08-1fda-4aab-90e7-1afc4d0c0118.png)

**In about;cache:**

![2](https://user-images.githubusercontent.com/65161029/194726086-7c67cf91-f5b6-4aaf-ba63-8abeb11d14c2.png)

**In DNSCrypt-Proxy query log:**

![3](https://user-images.githubusercontent.com/65161029/194726113-1ee92165-cd27-48ff-b0f6-ab13ef853775.png)

`about:debugging#/runtime/this-firefox` > `Violentmonkey` **inspect/Console** is empty, but before that the about debugging displays two **_warnings_**:

![4](https://user-images.githubusercontent.com/65161029/194726278-641e13af-3520-4b4b-b97f-e00efa912fa4.png)

From there on I remain stunned. None of this with previous VM 2.13.1, yet you cannot reproduce. Stunned imaculate :=)

**Comment by tophf** [comment]
I've asked you to click the `Inspect` button and check the console inside, you didn't. You don't need to prove anything to me, just work with me toward solving the problem.

**Comment by tophf** [comment]
Ah, sorry, you did click it but then showed a screenshot that I didn't ask. Well, I have no more ideas at the moment. Will post later.

**Comment by tophf** [comment]
So far it seems my initial guess is correct: it's not the auto-update per se, but a check for a missing resource, which is incorrectly triggered even though the script evidently exists, otherwise it wouldn't work.

**Comment by Cade66** [comment]
I have no idea, @tophf -- I'm not trying to prove whatever but only to add to your testing requirements (checking console in about:debugging) any further information that may come into consideration. I guess the warnings provided interesting information.

There's always an explanation, but the first challenge is to know if it's specific to the user or to the application, which is why I insisted on the fact that previous VM 2.13.1 did not encounter this issue on my device.

Of course, thanks for your work, truly most appreciated. I've been using Violentmonkey since always, with more or less downloaded scripts (I used to have many many more before cleaning up) and several home-made ones, always flawlessly.
